### PR TITLE
Create WebDev3.html

### PR DESCRIPTION
**Pull request by Bsavard35**
Changed latitude and longitude to show duluth and also changed zoom to 12.

### CODE REVIEW

**Comment by Bsavard35** [comment]
Could not figure out how to get step 5 done as I spent an hour total just on that step and could not figure it out still. even messaged a few people from class.

**Comment by kristofj-umd** [comment]
> Could not figure out how to get step 5 done as I spent an hour total just on that step and could not figure it out still. even messaged a few people from class.

Thanks for the comment. I'm glad you gave it a go, and then gave it up after a good effort.

File: WebDev3.html [hunk 1]
```diff
@@ -94,19 +94,40 @@ <h5 class="right">GIS4533 | Fall 2020 | Ben Savard</h5>
...
     // Create variable to hold map element, give initial settings to map
     var map = L.map('map',{ center: [46.7867, -92.1005], zoom: 12}); 
-          
+         
+           Source: http://leaflet-extras.github.io/leaflet-providers/preview/ 
+           var Thunderforest_TransportDark = L.tileLayer('https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png?apikey={apikey}', {
+	attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+	apikey:  'd8ebd0005b0d499b9953b6859696b681',
+	maxZoom: 22
+}).addTo(map);
```
Comment (kristofj-umd) [inline WebDev3.html:103]: I like this one a lot. 😎 
Did you have to create your own API key in order to use it?

File: WebDev3.html [hunk 2]
```diff
@@ -94,19 +94,40 @@ <h5 class="right">GIS4533 | Fall 2020 | Ben Savard</h5>
...
+	attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+	apikey:  'd8ebd0005b0d499b9953b6859696b681',
+	maxZoom: 22
+}).addTo(map); 
            
           // Add OpenStreetMap tile layer to map element
-L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '© OpenStreetMap' }).addTo(map);
 omnivore.csv('airports.csv').addTo(map); 
-           
+           Source: http://duspviz.mit.edu/web-map-workshop/leaflet-js/
```
Comment (kristofj-umd) [inline WebDev3.html:107]: I think you are missing the `//` comment characters here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          
+          // Create line feature for Mass Ave Bridge, style and add to map
+var myDataLine = L.polyline([[42.357227, -71.092631], [42.351411, -71.089723]],
+    {color: 'red', weight: 10}).addTo(map); 
+          
+          
+          // Create area feature for MIT, style and add to map
+var myArea = L.polygon([[42.353770, -71.103606], [42.355447, -71.104475],
+    [42.362681, -71.089830], [42.361829, -71.079230]],
+    {color: 'blue', weight: 4}).addTo(map);
```
Comment (kristofj-umd) [inline index.html:35]: To my understanding the changes to your code need only include the above portion. So why are committing this whole file in a single commit?

**Comment by kristofj-umd** [comment]
Nice work overall. It looks you attempted all of the tasks even if you did not complete them.

**Review by kristofj-umd — COMMENTED** [review]
I like the verbose descriptions provided in your commit summaries.

Nice work overall. It looks you attempted all of the tasks even if you did not complete them.

**Comment by Bsavard35** [comment]
I think I made that commit cause it was giving me some error commit message on GitHub desktop so I had to make a commit of the full thing it was weird but I went with it. 

Yes I believe I did hVe to make my own api key which was not to hard to figure out!!!!

**Pull request merged**